src/models/Alert.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * LINKURIOUS CONFIDENTIAL
+ * Copyright Linkurious SAS 2012 - 2019
+ *
+ * - Created on 2019-05-16.
+ */
+
+// TS2019-DONE
+
+import {FolderChildren, IDataSourceParams, IPersistedItem} from './Model';
+
+export interface ICreateAlertFolderParams extends IDataSourceParams {
+  title: string;
+  parent?: number;
+}
+
+export interface ICreateAlertFolderResponse extends IPersistedItem {
```

Comment:
why do we need the sourceKey in the response ?

Reply:
we don't indeed